packages/node/rollup.config.js

```diff
@@ -50,7 +50,6 @@ export default [
         globals: globals
       }
     ],
-    sourceMap: true,
```

Comment:
Where did you see rollup not recognizing this?

According to its docs, it does: https://rollupjs.org/guide/en#big-list-of-options

Reply:
That's only if its inside one of the things inside the `output` key, not in the top level

Reply:
And I saw this in the output of running `yarn build`

```
⚙️  Building package: node
$ rollup -c
(!) You have passed an unrecognized option
Unknown input option: sourceMap. Allowed options: acorn, acornInjectPlugins, cache, chunkGroupingSize, context, experimentalCacheExpiry, experimentalOptimizeChunks, experimentalTopLevelAwait, external, inlineDynamicImports, input, manualChunks, moduleContext, onwarn, perf, plugins, preserveModules, preserveSymlinks, shimMissingExports, treeshake, watch
```

Reply:
Then move it to `output`?

Reply:
It is already there